examples/mux-selftest.yaml

```diff
@@ -7,16 +7,32 @@
 #          multiple files and checks that the node ordering works fine.
 # /env/opt_CFLAGS: Should be present in merged node
 # /env/prod/opt_CFLAGS: value should be overridden by latter node
+# The internal filters are designed to be used for this file injected into
+# /virt (use -m /virt:examples/mux-selftest.py). When it's injected into
+# a different location those filters should not affect the result (produces
+# all variants.
+# !filter-only: All root childern are specified in different levels. They
+#               should be combined and together enable all variants. On the
+#               other hand they should not enable other-level filter-only
+#               filters like /hw/disk/virtio.
+
 hw:
+    # This filter has no effect, it's here to test filter inheritance
+    !filter-out : /this/does/not/exists
```

Comment:
No reason to hate the file, it was developed when `examples` was widely used for selftests... Anyway here is the card https://trello.com/c/DpplIF8A/983-move-mux-selftest-from-examples-to-selftests-data-and-create-few-examples